src/py/covidr/config/config.py

```diff
@@ -0,0 +1,4 @@
+class Config(object):
+    DEBUG = False
+    TESTING = False
+    DATABASE_URI = ""
```

Comment:
We'll have to reconcile this and https://github.com/chanzuckerberg/covidr/pull/8/files#diff-e21eb535636415b0f95d7d87b7e9f5164945ec1ff0f00116d547b2028273b261.  Proposal: let's land both as is, and we can refactor.